engine/src/main/java/org/hibernate/validator/internal/engine/cascading/ObservableValueExtractor.java

```diff
@@ -19,6 +20,7 @@
  * @author Gunnar Morling
  */
 @IgnoreJava8Requirement
```

Comment:
No, it can't. I remember trying to remove it but animal-sniffer complains as JavaFX is not part of the Java 8 signatures.